TTS/bin/compute_embeddings.py

```diff
@@ -1,80 +1,41 @@
+
 import argparse
-import glob
 import os
 
-import torch
 from tqdm import tqdm
 
-from TTS.config import BaseDatasetConfig, load_config
-from TTS.speaker_encoder.utils.generic_utils import setup_model
+from TTS.config import load_config
 from TTS.tts.datasets import load_meta_data
 from TTS.tts.utils.speakers import SpeakerManager
-from TTS.utils.audio import AudioProcessor
+
```

Comment:
Can you add an example call as docstring here so that people know how to call it?

Reply:
I did this, but the test failed on python3.6.  Any idea how to solve it?

Reply:
let me run the test again